src/ghci/manager.rs

```diff
@@ -56,6 +56,7 @@ pub async fn run_ghci(
     // This function is pretty tricky! We need to handle shutdowns at each stage, and the process
     // is a little different each time, so the `select!`s can't be consolidated.
 
+    let no_interrupt_reloads = opts.no_interrupt_reloads;
```

Comment:
This definitely feels awkward. I tried passing the `opts` directly in to the relevant function, but the `opts` are moved into `Ghci::new` call just below. I can get the `bool` out here no problem, though, so I do that instead.

Reply:
Yeah I think this is fine.